components/studio/test/shared/studio-internals/test-studio-metadata.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash 
+
+set -euo pipefail
+
+exit_code=0
+
+if [ ! -f /.studio ]; then 
+  echo "Unable to find .studio metadata"
+  exit_code=1
+fi
+
+if ! grep -q 'studio_type="default"' /.studio; then 
+  echo "Studio type is not 'default'"
+  exit_code=1
+fi 
+
+if ! grep -q 'studio_path="/hab/bin"' /.studio; then 
+  echo "Studio path should be /hab/bin"
+  exit_code=1
+fi
+
+if ! grep -q 'studio_build_environment=""' /.studio; then 
+  echo "Studio build environment incorrect" 
+  exit_code=1
+fi
+
+if ! grep -q 'studio_run_environment=""' /.studio; then 
+  echo "Studio run environment incorrect" 
+  exit_code=1
+fi
```

Comment:
You need an `exit ${exit_code}` down here.

Reply:
👍  I never finished that test as I wasn't sure if it was actually useful.  I was originally worried about accidentally changing the contents of the file as part of my investigations into #6509 , but I don't know that we actually use that file anywhere.